tools/run_tests/run_tests_matrix.py

```diff
@@ -246,10 +246,10 @@ def _create_test_jobs(extra_args=[], inner_jobs=_DEFAULT_INNER_JOBS):
     # supported on mac only.
     test_jobs += _generate_jobs(
         languages=['objc'],
-        configs=['dbg', 'opt'],
+        configs=['dbg'],
         platforms=['macos'],
         labels=['basictests', 'multilang'],
-        extra_args=extra_args,
+        extra_args=extra_args + [os.getenv('GRPC_OBJC_TEST_EXTRA_ARGS', '.*')],
```

Comment:
passing a value of an env variable like this seems arbitrary and cryptic.

Reply:
The reason for using extra args is because the `ObjCLanguage` object in `run_tests.py` does not have an option of target platform. The platform information in `run_tests.py` are all inferred from platform.system. I did not consider adding an option to `run_tests_matrix.py` or `run_tests.py` before, but if that is allowed then I can tweak the environment variable part into run options.